### ISSUE TITLE
Cosmetic: Move dialogue is flawed

### ISSUE THREAD

**Issue by karlic** [kolyvan/kybook#67]
KyBook v0.5.5 | #99

1. Long tap a book
2. Tap `iPad`
3. The Body of the dialogue says, "Move X file from iPad//source_folder"
4. The bottom of the dialogue says, "Select a folder"
5. The top-left of the dialogue says, "< Select a folder"

The text at 3. could say, "Move X file from source_device//source_folder to destination_device//destination_folder"

The text at 4. should say, "Move here". The current wording is wrong (you can't select a folder by tapping `Select a folder`) and is confusing as it duplicates the text at 5 with different functionality.

NOTE also that if at 1. you long tap a **_folder_**, the text at 3. says "Move 1 file from iPad//", but:
A. it's a folder not a file
B. it would be better if it said, "Move X file(s) in 1 folder ..."

I notice:
i) Copy (as an alternative to Move) has been removed
ii) it is not possible to move more than one folder at a time
I assume i) and ii) are by design.

**Comment by karlic** [comment]
PS. If you do change the above, you might need to change the pop-up that appears after the move has completed.

**Comment by kolyvan** [comment]
> The text at 3. could say, "Move X file from source_device//source_folder to destination_device//destination_folder"

We can't say that.
The destination folder is still unknown till a user taps on `Select a folder` button. 

> The text at 4. should say, "Move here". 

Maybe, but it's a general dialog, the title of button resused and in other scenarios.
I will think about fixing a title.

> it's a folder not a file

In fact everything is a file, even a folder. 
Also an user can select files and folders at the same time.
But yes, user may be confused.

**Comment by kolyvan** [comment]
> Copy (as an alternative to Move) has been removed

No, you can change `Move` dialog to `Copy` mode when your a source or destination is the Cloud Storage. Copying between the device folders has no sense.

**Comment by kolyvan** [comment]
> it is not possible to move more than one folder at a time

Please, KyBook is not a file manager at all!!

**Comment by karlic** [comment]
> Please, KyBook is not a file manager at all!!

Absolutely agree, just wanted to check that was the design. :-)

> We can't say that. The destination folder is still unknown till a user taps on Select a folder button.

OK, I thought an event would get raised when the user tapped a folder and you could update the display at that point. It's not a big deal, just gives the user confidence that the files are going where she thinks they're going.
